Source/Analytics/Read/Map/CaseReportsBeforeDay.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using Dolittle.ReadModels;
+using Concepts;
+using Concepts.HealthRisk;
+using System.Collections.Generic;
+using MongoDB.Bson.Serialization.Attributes;
+using MongoDB.Bson.Serialization.Options;
+
+namespace Read.Map
+{
+    public class CaseReportsBeforeDay : IReadModel
```

Comment:
I'm not sure I understand the naming here... To me, this name indicates that you want all case reports before a given day. However, it is grouped by health risk – maybe include that in the name in some way?

Reply:
Comment from Gard on other PR: "I'm a bit confused by the name here, what about something like CaseReportsForMapPerDay ?"